docs/features/logforwarding/outputs/lokistack-forwarding.adoc

```diff
@@ -0,0 +1,450 @@
+:toc:
+:toclevels: 2
+
+= Forwarding To RH Managed LokiStack
```

Comment:
This fundamentally describes standing up a complete, managed logging solution including all components.  I think we may should consider moving this to the "administration" section since its a "give me everything" document

Reply:
Yes, that makes more sense.   I could carve out just the forwarder config for this doc and move the "complete" doc to administration.

Reply:
I've moved this to administration dir and called it 'lokistack' ?    I carved out only the CLF sections for the forwarding feature doc.